lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -97,15 +97,37 @@ sub group_overview {
     my @pinned_comments;
     my $tags;
     if ($group->can('comments')) {
-        for my $comment ($group->comments->all) {
-            # find pinned comments
-            if ($comment->user->is_operator && CORE::index($comment->text, 'pinned-description') >= 0) {
+        # find comments
+        my $page               = int($self->param('comments_page') // 1) // 1;
+        my $comments_resultset = $self->app->schema->resultset('Comments');
+        my $comments           = $comments_resultset->search(
+            {
+                group_id => $group->id
+            },
+            {
+                page     => $page,
+                rows     => 5,
```

Comment:
the 5 is a big magic - and repeated here and in the template. Better pass it as param from the template and use a variable there.

Reply:
True. I have to change that code anyways because having some "..." in the gap would be nice.